phpBB/includes/ucp/ucp_register.php

```diff
@@ -630,10 +630,10 @@ function main($id, $mode)
 		phpbb_timezone_select($template, $user, $data['tz'], true);
 
 		// Checking amount of available languages
-		$sql = 'SELECT lang_id
-			FROM ' . LANG_TABLE;
+		$sql = 'SELECT lang_iso, lang_local_name
+			FROM ' . LANG_TABLE . '
+			ORDER BY lang_english_name';
```

Comment:
The old query should also work, because when you delete a language the whole row for it is removed from the LANG_TABLE but still remains in the filesystem so that the administrator can install it again. So counting IDs is perfectly fine, the default EN language pack cannot be removed at all.

Reply:
It's used as an argument for `language_select()` so should match its data format.

Reply:
Yeah a different approach than the one done already in `ucp_prefs.php`

Reply:
So there are 2 logics behind the same type of result, now.